### DISCUSSION TITLE
Versão 16 - Erro ao criar novo  lancamento no diario

### DISCUSSION THREAD

**Discussion by gilsonlima10** [OCA/l10n-brazil#3562] [Q&A]
UncaughtPromiseError

Promessa não capturada > for modifier "readonly": Unknown field product_id in domain

Error: for modifier "readonly": Unknown field product_id in domain
    _evalModifiers@http://10.0.0.20:8069/web/assets/521-084d37c/web.assets_backend.min.js:8161:138
    _isFieldProtected@http://10.0.0.20:8069/web/assets/521-084d37c/web.assets_backend.min.js:8242:361
    _generateChanges@http://10.0.0.20:8069/web/assets/521-084d37c/web.assets_backend.min.js:8187:321
    _save@http://10.0.0.20:8069/web/assets/521-084d37c/web.assets_backend.min.js:8093:18
    exec/this.lock</<@http://10.0.0.20:8069/web/assets/512-20bc6d2/web.assets_common.min.js:5797:607

**Comment by antoniospneto** [comment]
@gilsonlima10 

O local correto para reportar um problema envolvendo diretamente a localização, é na aba Issues: https://github.com/OCA/l10n-brazil/issues

Mas testei e realmente isso é um problema:
![image](https://github.com/user-attachments/assets/157d758b-2fc0-46ce-aff6-8fc7bd7c92ee)

**Comment by antoniospneto** [comment]
Aberto a issue #3573

**Comment by rvalyi** [comment]
resolvido pelo PR #3576
